tools/UpdatePaths/Scripts/90094_Preloaded_pacmans_to_spawners.txt

```diff
@@ -0,0 +1 @@
+/obj/machinery/power/port_gen/pacman/pre_loaded : /obj/effect/spawner/random/engineering/pacman {@OLD}
```

Comment:
This is not what you use UpdatePaths for - this will prevent mappers from pre-mapping pacmans, which is bad. UpdatePaths should be used exclusively for repaths, i.e. when the previous path is deleted.

Reply:
Will remove it later today